app/models/shift.rb

```diff
@@ -48,4 +49,13 @@ def shift_ends_after_start
     end
   end
 
+  def shift_start_before_current_date?
+    return if self.shift_start.blank? || self.shift_end.blank?
```

Comment:
Hey @GabrielleEaston since we repeat this line in three private methods, let's DRY up the code by creating a private method called `any_shift_time_blank?` and then calling in on lines 37, 45 and 53 👍🏾

Reply:
@msarit Thank you for pointing it out! Resolved  it! Method added!